android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/QuestionUiModel.kt

```diff
@@ -1,5 +1,13 @@
 package vn.luongvo.kmm.survey.android.ui.screens.survey
 
+import vn.luongvo.kmm.survey.domain.model.Question
+
 data class QuestionUiModel(
-    val text: String
+    val text: String,
+    val coverImageUrl: String
+)
+
+fun Question.toUiModel() = QuestionUiModel(
+    text = text.orEmpty(),
+    coverImageUrl = coverImageUrl.orEmpty() + "l"
```

Comment:
optional: I think `coverImageUrl` should be a computed variable `largeCoverImageUrl`, we can keep `coverImageUrl` as is for smaller image (we can use for low internet mode) and have largeCoverImageUrl be `coverImageUrl + "l"`.

Reply:
@blyscuit possible but it's not clear to be an AC in the IC. I would like to always show high-quality images for the background. Having low internet support is out of scope (and I will make the renaming in case we need it instead).